tests/unit/lib/SGVector_unittest.cc

```diff
@@ -193,9 +170,9 @@ TEST(SGVectorTest,complex128_tests)
 	SGVector<complex128_t> a(10);
 	a.set_const(complex128_t(5.0, 6.0));
 	SGVector<complex128_t> b=a.clone();
-
+	
 	// test ::operator+ and []
-	a=a+b;
+	a=linalg::add(a,b);
```

Comment:
i think we would just remove the addition tests of SGVector ... linalg::add is tested already

Reply:
But not for complex numbers though?

Reply:
sure, but then we would have a test for linalg, and not sgvector, this is not testing sgvector::operator+ anymore, or am I wrong?

Reply:
Indeed, that’s the other option I suggested above

Reply:
So what do you suggest we do here?
Should I simply remove the test and open another issue?

Reply:
Can you go to `Eigen3_operations_unittest.cc`, create a new class inheriting from ::testing::Test, then pass it to `SG_TYPED_TEST_CASE(..., sg_all_primitive_types, bool);` and use that new class in `TYPED_TEST` where we test for `add`. Then you can delete this SGVector test :)

Reply:
And give that class a meaningful name like the other ones in `Eigen3_operations_unittest.cc` please!

Reply:
Yes I will do that. However, I have a doubt, Doesn't the existing code to that currently as well? reference
line 26 
```
template <typename T>
class LinalgBackendEigenAllTypesTest : public ::testing::Test
{
};
```

line 43
```
SG_TYPED_TEST_CASE(
    LinalgBackendEigenAllTypesTest, sg_all_primitive_types, bool, complex128_t);
```
line 50
```
TYPED_TEST(LinalgBackendEigenAllTypesTest, SGVector_add)
{
```

Reply:
Unfortunately it doesn't do all the types like it says... You can see on line 43 that bool and complex128_t are excluded from the types being test